modyn/evaluator/internal/grpc/evaluator_grpc_servicer.py

```diff
@@ -100,12 +101,16 @@ def evaluate_model(self, request: EvaluateModelRequest, context: grpc.ServicerCo
 
             if not trained_model:
                 logger.error(f"Trained model {request.model_id} does not exist!")
-                return EvaluateModelResponse(evaluation_started=False)
+                return EvaluateModelResponse(
+                    evaluation_started=False, not_start_reason=EvaluationNotStartReason.MODEL_NOT_EXIST_IN_METADATA
```

Comment:
Similar here, depending on what name you decide for, can we rename the argument to `not_started_reason` or `failure_reason` or `cancel_reason` or sth like this please? Thanks